packages/celotool/src/cmds/geth/start.ts

```diff
@@ -103,6 +106,21 @@ export const builder = (argv: yargs.Argv) => {
       description: 'Block Time',
       default: 1,
     })
+    .option('migrate', {
+      type: 'boolean',
+      description: 'Migrate contracts',
+      default: false,
+      implies: 'monorepo-dir',
+    })
+    .option('migrateTo', {
+      type: 'number',
+      description: 'Migrate contracts to level x',
+      implies: 'monorepo-dir',
+    })
+    .option('monorepo-dir', {
+      type: 'string',
+      description: 'Directory of the mono repo',
```

Comment:
Useful to have as an option. I imagine most users will want to use the current monorepo directory, so making that the default would be good.

Reply:
it is only relevant when using the migrate option so it will be a special case. But i can of course set a default.